.hlint.yaml

```diff
@@ -14,7 +14,6 @@
 - ignore: {name: "Use fold"} # 1 hint
 - ignore: {name: "Use gets"} # 9 hints
 - ignore: {name: "Use guards"} # 3 hints
-- ignore: {name: "Use infix"} # 4 hints
```

Comment:
This one I could leave ignored. I'd rather not bother people to use some operators in infix style. But I don't feel too strong about it. Does anyone else want to comment?